.github/workflows/reusable-testing.yml

```diff
@@ -45,14 +45,6 @@ jobs:
         if: inputs.use-valgrind
         run: sudo apt update && sudo apt install -y valgrind
 
-      # https://github.com/actions/runner-images/issues/8659
```

Comment:
Can you please drop a comment in the commit message about what happened?

Reply:
Done.